src/rules/incognia-onboarding.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @title Incognia Onboarding Rule
+ * @overview Verify if the device location behavior matches the address declared during onboarding.
+ * @gallery true
+ * @category marketplace
+ *
+ */
+async function incogniaOnboardingRule(user, context, callback) {
+  const _ = require('lodash@4.17.19')
+
+  const { IncogniaAPI } = require('@incognia/api@1.0.0')
+  const { Auth0UserUpdateUtilities } = require('@auth0/rule-utilities@0.2.0')
+
+  const { INCOGNIA_CLIENT_ID, INCOGNIA_CLIENT_SECRET } = configuration;
```

Comment:
Same question as the other Rule:

https://github.com/auth0/rules/pull/276#discussion_r598768578

Reply:
Yes! Same credentials can be reused!